media/stylus/components/wiki/page-buttons.styl

```diff
@@ -34,6 +34,13 @@
     }
 }
 
+/* Users can style zone landing pages. We can't be sure that dark blue will look
+   good on background colors they choose. */
+body:not(.zone-landing) #edit-button {
```

Comment:
Ah. Gotcha.

I spent some time thinking about that. We could alternatively give non-landing pages a new class. Something like `.standard-article`. But then if we create a third category of page later on, selecting `.standard-article` won't be enough.

We ultimately want to select everything that *isn't* a zone landing page, so I think the `:not` selector is a natural fit.